.goreleaser.yml

```diff
@@ -0,0 +1,57 @@
+project_name: nats-server
+
+release:
+  github:
+    owner: nats-io
+    name: nats-server
+  name_template: 'Release {{.Tag}}'
+  draft: true
+
+changelog:
+  skip: true
+
+builds:
+- main: ./main.go
+  binary: nats-server
+  ldflags:
+    - -s -w -X github.com/nats-io/nats-server/server.gitCommit={{.ShortCommit}}
+  env:
+    - GO111MODULE=off
+    - CGO_ENABLED=0
+  goos:
+  - darwin
+  - linux
+  - windows
+  goarch:
+  - amd64
+  - arm
+  - arm64
+  - 386
+  goarm:
+  - 6
+  - 7
+  ignore:
+    - goos: darwin
```

Comment:
What does this do since we list darwin above?

Reply:
This is excluding a combination of os and arch. For instance this means to ignore darwin 386.